sheetstorm/management/commands/generate_test_data.py

```diff
@@ -186,3 +223,27 @@ def add_arguments(self, parser: Any) -> None:
             type=int,
             help="Indicates the maximum number of completed projects to be in the database",
         )
+        parser.add_argument(
+            "--small",
+            dest=TestDataSize.SMALL.name,
+            action="store_true",
+            help="Use prepared small set to generate test data",
+        )
+        parser.add_argument(
+            "--medium",
+            dest=TestDataSize.MEDIUM.name,
+            action="store_true",
+            help="Use prepared medium set to generate test data",
+        )
+        parser.add_argument(
+            "--large",
+            dest=TestDataSize.LARGE.name,
+            action="store_true",
+            help="Use prepared large set to generate test data",
+        )
+        parser.add_argument(
+            "--extra_large",
```

Comment:
I didn't noticed it at first time. When you creating parameters should like this which will be used in bash it should be divided by "-" not by "_" because in bash is convention that words should be split with dash